logging/subscriber/src/subscriber.ts

```diff
@@ -0,0 +1,34 @@
+import type { EventPayload, Layer } from '@rango-dev/logging-types';
+
+import { EventType, Level } from '@rango-dev/logging-types';
+
+import { isEnabled } from './helpers';
+
+interface Options {
+  maxLevel: Level;
+}
+
+const DEFAULT_LEVEL = Level.Trace;
```

Comment:
It is better to change the **DEFAULT_LEVEL** to cover more events

Reply:
`event.level` always should have a valid structure and having `level`, so this fallback was redundant. I removed it and added a runtime check.